### PR TITLE
Fix unit test for smart contract

### PR DESCRIPTION
**Pull request by juniuszhou**

### CODE REVIEW

File: smart-contracts/contracts/CosmosBridge.sol [hunk 1]
```diff
@@ -226,13 +227,16 @@ contract CosmosBridge is CosmosBridgeStorage, Oracle {
         pow += getValidatorPower(validator.signer);
       }
 
-      // Signatures must be sorted on the sifchain side, so we just
-      // need to make sure that the next validator in the array
+      // Signatures must be sorted on the relayer side, so we just
+      // need to make sure that the next witness in the array
```
Comment (jzvikart) [inline smart-contracts/contracts/CosmosBridge.sol:231]: What is the actual sorting key? (i.e. `validator.signer`)

Comment (juniuszhou) [reply smart-contracts/contracts/CosmosBridge.sol:231]: sorting key is signer, cosmosBridge need check if the duplicated signer in the signature list. sorted singer make it efficient to compare, gas saving.

**Comment by jzvikart** [comment]
@Brando753 Shall we also add a test to check for missing `bridgeBank.addExistingBridgeToken()`?

**Comment by juniuszhou** [comment]
we may even remove it, if don't need to do such operation in the future. all bridge tokens will be created and put into whitelist automatically.

**Comment by jzvikart** [comment]
Looks good to me!

**Review by jzvikart — APPROVED** [review]

**Comment by juniuszhou** [comment]
#2793 created to discuss the addExistingBridgeToken method.

**Pull request merged**